README.md

```diff
@@ -1,24 +1,22 @@
 # Exercism's Go Test Runner
 
-This is Exercism's test runner for the Go track.
+This is [Exercism's test runner](https://github.com/exercism/v3-docs/tree/master/anatomy/track-tooling/test-runners#test-runners) for the Go track.
 
 ## Executing the Test Runner
 
-The test runner takes 3 parameters:
-- the exercise `slug`, e.g. `two-fer`
-- the `path` containing the solution to test
-- the `output path` for the test results
+The test runner takes 2 parameters:
+- `input_dir`: the path containing the solution to test
+- `output_dir`: the output path for the test results
```

Comment:
You're inferring the exercise slug from the directory name? If so, this will be problematic when the test runner executes in our production environment, as the directory names will _not_ match the exercise name. I would suggest to take the slug as the first parameter. The Docker command listed below does have this slug:

```bash
docker run -v ~/Exercism/go/gigasecond:/solution exercism/go-test-runner gigasecond /solution /solutio
```

Reply:
That's what I mean with "new rules". From the code it looked to me like `slug` was deprecated. Turns out it wasn't.

Reply:
Correct!

Reply:
I didn't realize the test runner was not currently correct - I thought the only task was to add the `test_code` field.  I can definitely get the output up to spec. Referencing [the comment I just posted below](https://github.com/exercism/go-test-runner/pull/17#issuecomment-740664545), in order to focus on one fix at a time, I'd like to separate this fix into a separate issue / PR. Is that acceptable,  @ErikSchierboom?

Reply:
This test runner was one of the first three made when we were really prototying things last year. So there have been a few small changes since then, but I think it's probably still very close to being "correct", bar the `test_code` field. So hopefully not **too** much work!

Reply:
For sure - getting it up to spec should be a snap compared to extracting the `test_code`. If we can get this PR merged, I'll get the output up to spec in #19 (which is branched off of this branch), or a future PR.